tests/e2e/pageobjects/ide/ProjectTree.ts

```diff
@@ -156,7 +156,7 @@ export class ProjectTree {
                 return true;
             } catch (err) {
                 if (!(err instanceof error.TimeoutError)) {
-                    throw err('Unexpected error during project tree expanding');
+                    throw Error('Unexpected error during project tree expanding');
```

Comment:
I would delete message at all
```
throw err;
```

To show the original error log.
It may be helpful for investigation if it happens.

Reply:
Got it. Will try.

Reply:
Done.